test_files/values.alp

```diff
@@ -15,6 +18,20 @@ let test_atom = :nope
 let test_bool = true
 let test_chars = c"x"
 let test_tuple = (100, "100")
+let test_unit = ()
+
+let wrapped_int = AnInt 41
```

Comment:
Or rather allow symbols as literal values (literal is not as fitting a name then, perhaps), and then have the typer or ast_gen stage complain if the symbol doesn't refer to a zero arity function.

Constructors as literals are already supported. as the test witnesses.